src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
```

Comment:
Avoid using @author tag as it is considered outdated and not recommended. (Severity: low)